kythe/go/serving/pipeline/beamio/leveldb_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package beamio
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"sort"
+	"testing"
+	"unicode"
+
+	"github.com/apache/beam/sdks/go/pkg/beam"
+	"github.com/apache/beam/sdks/go/pkg/beam/testing/ptest"
+	"github.com/jmhodges/levigo"
+
+	_ "github.com/apache/beam/sdks/go/pkg/beam/io/filesystem/local"
+)
+
+func TestLevelDBSink(t *testing.T) {
+	var strs []string
+	for i := 0; i < 28; i++ {
+		buf := ""
+		for c := 'A'; c <= 'z'; c++ {
+			if !unicode.IsLetter(c) {
```

Comment:
are there a lot of characters in between A and z that aren't letters?  that is, does this actually gain us anything?

Reply:
I wanted the keys to be completely readable for test failures.